tests/unit/utils/test_http.py

```diff
@@ -134,3 +141,43 @@ def test_requests_multipart_formdata_post(self):
         body = ret.get('body', '')
         boundary = body[:body.find('\r')]
         self.assertEqual(body, match_this.format(boundary))
+
+    @skipIf(HAS_CURL is False, "Missing prerequisites for tornado.curl_httpclient library")
```

Comment:
I wonder if we can ensure that it will be triggered on some CIs. Right now in a case if there is no pycurl installed on your Jenkins - this test will be never triggered and nobody will even notice that.

Reply:
we are ensuring that we install pycurl by adding it as a requirement in `requirements/static/linux.in ` which nox pulls from. The tests failed when attempting to install pycurl, so once libcurl-dev is installed the tests will run on linux distros for sure.

Reply:
oh right, thanks for the clarification!